.buildkite/pipeline.graphql.yml

```diff
@@ -0,0 +1,13 @@
+steps:
+  - label: Update GraphQL docs
+    command: .buildkite/update_graphql_docs
+    plugins:
+      - docker-compose#v4.9.0:
+          run: app
+          mount-ssh-agent: true
+          mount-buildkite-agent: true
+          env:
+            - API_ACCESS_TOKEN
+            - GIT_NAME
+            - GIT_EMAIL
+            - GH_TOKEN
```

Comment:
Looks like you're not using `GH_TOKEN` anywhere?

Reply:
Good spot! `GH_TOKEN` is a [special env variable](https://cli.github.com/manual/gh_help_environment) `gh-cli` uses for authentication without needing to call the auth APIs.

> GH_TOKEN, GITHUB_TOKEN (in order of precedence): an authentication token for github.com API requests. Setting this avoids being prompted to authenticate and takes precedence over previously stored credentials.